jme3-alloc/src/main/java/com/jme3/alloc/util/loader/NativeBinaryLoader.java

```diff
@@ -46,13 +49,30 @@
  */
 public final class NativeBinaryLoader {
     
+    private static final Logger LOGGER = Logger.getLogger(NativeBinaryLoader.class.getName());
     private static final ReentrantLock LOCK = new ReentrantLock();
     private static final int EOF = -1;
     private static boolean autoLoad = true;
 
     private NativeBinaryLoader() {
     }
 
+    /**
+     * Extracts and loads the variant specific binary from the output jar, handling the error messages, 
+     * guarded by the {@link NativeBinaryLoader#isAutoLoad()}.
+     */
+    public static void quickLoadLibrary() {
```

Comment:
What about naming it `autoLoadLibrary()`?

Reply:
I haven't chosen this name; because technically saying, the static initializer of NativeBufferUtils and this method produce the autoLoading, so using this method alone will not be efficient to name the process as autoLoad.

I am thinking now of `guardedLoadLibrary` over `quickLoadLibrary`, what do you think? Do you have other suggestions?

Reply:
IMO we should modify `loadLibrary` to just log a warning when autoLoad is off and return imidiatly.

Reply:
and rename `NativeBinaryLoader.setAutoLoad()` to `NativeBinaryLoader.setEnabled()`.

Reply:
Good, that would mean removing this method completely.

Reply:
Okay, here is what I am going to do about this in another PR: 
1) Rename `quickLoadLibrary` to `loadLibraryIfEnabled`.
2) Rename `autoLoad` to `enabled`, so you will be using `NativeBinaryLoader.setEnabled(false)`.

I think that way, we will resolve the confusing names.

Reply:
>loadLibraryIfEnabled

This sounds unnecessary to me. I think there should be only one load method.

Reply:
I would like to keep 2 methods, in case one would need to catch the exceptions out of it.

Reply:
I will keep the `autoLoad` to `enabled` refactor for another PR.

Reply:
> Edit:
> You should move the try-catch block to the calling class.

I will think of this, but this will result in code duplication of NativeErrno and NativeBufferUtils, and also the NativeBinaryLoader logger will be called outside its scope...

Reply:
Looking at JME, it throws UncheckedIOException internally inside the load function

https://github.com/jMonkeyEngine/jmonkeyengine/blob/44a50de4d7797dc3dd8d46cd1f51308e9a05d627/jme3-desktop/src/main/java/com/jme3/system/NativeLibraryLoader.java#L526-L534

Maybe you should do a similar thing.